app/lib/data_html_parser.rb

```diff
@@ -88,6 +88,30 @@ def get_clustermapping_series(dataset, parameters)
     new_data
   end
 
+  def get_eia_series(parameter)
+    api_key = ENV['API_KEY_EIA']
+    raise 'No API key defined for EIA' unless api_key
+    @url = "http://api.eia.gov/series/?series_id=#{parameter}&api_key=#{api_key}"
```

Comment:
Although there is a redirect in place, might be better to have `https` directly in the code :=)